### PR TITLE
Remove window.dataExplorerPlatform

### PR DESCRIPTION
**Pull request by southpolesteve**
More cleanup for #253 
- Remove window.dataExplorerPlatform
- Remove explorer factories

### CODE REVIEW

File: src/Explorer/Controls/Settings/SettingsSubComponents/__snapshots__/ScaleComponent.test.tsx.snap [hunk 1]
```diff
@@ -39,13 +39,13 @@ exports[`ScaleComponent renders with correct intiial notification 1`] = `
     }
   >
     <ThroughputInputAutoPilotV3Component
-      canExceedMaximumValue={false}
+      canExceedMaximumValue={true}
       getThroughputWarningMessage={[Function]}
```
Comment (languy) [inline src/Explorer/Controls/Settings/SettingsSubComponents/__snapshots__/ScaleComponent.test.tsx.snap:43]: Just pointing out that there's a functional change here and line 48 (the description doesn't mention it) and making sure it's intended.

**Review by languy — APPROVED** [review]

File: src/Explorer/Controls/Settings/SettingsSubComponents/__snapshots__/ScaleComponent.test.tsx.snap (hunk 1 above)
Comment (southpolesteve) [reply src/Explorer/Controls/Settings/SettingsSubComponents/__snapshots__/ScaleComponent.test.tsx.snap:43]: Yes. The change is the tests were running with dataExplorerPlatform === undefined but now we are pulling from configContext.platform which defaults to Portal

**Review by victor-meng — APPROVED** [review]

**Comment by gitanuj** [comment]
@southpolesteve This change would break `getDataExplorerWindow` in `WindowUtils` here
https://github.com/Azure/cosmos-explorer/blob/274deb13be2980d563cefd9c82cf544445c80177/src/Utils/WindowUtils.ts#L9

This in turn would break `sendMessage` in `MessageHandler` for windows other than Data Explorer (like Terminal). Is there any other way to identify DE window?

**Comment by southpolesteve** [comment]
@gitanuj I have opened up a fix here https://github.com/Azure/cosmos-explorer/pull/285/files

**Pull request merged**